### PR TITLE
Close #4 and #5: no particle in 3D example & erroneous basePath

### LINKED ISSUE #4: No particles in 3D example
There are no particles in the 3D example data set.

### PR DESCRIPTION
**Pull request by RemiLehe**
This pull requests loads new datasets:
- These datasets comply with the updated openPMD-validator (esp. the basePath error in Warp was corrected)
- Particles were added in the 3D example

For documentation and reproducibility, I also added the scripts that produce these datasets.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
Thanks @RemiLehe. I will have a look at the new data sets tomorrow.

**Comment by PrometheusPi** [comment]
Great job @RemiLehe. 
If it is fine for you and @ax3l that the `example-thetaMode` has no particles, this pull request can be merged. (I am no maintainer, so I can not merge.)

File: scripts/warp_thetaMode_script.py [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+v_moving_window = clight
+
+# Diagnostics
+# -----------
+# Period of diagnostics (in number of timesteps)
+diag_period = 100
+# Whether to write the fields
+write_fields = 1
+# Whether to write the particles
+write_particles = 0
```
Comment (PrometheusPi) [inline scripts/warp_thetaMode_script.py:66]: I would suggest setting this to `1`.

**Comment by ax3l** [comment]
@RemiLehe thank you a lot for the quick update and thanks to @PrometheusPi for testing!

**Comment by ax3l** [comment]
yes if @RemiLehe agrees we can also add particles in the `theaMode` sims.

**Comment by PrometheusPi** [comment]
**Warning:** You added one zero to the file names. Thus **old scripts** reading the data **will not work**.

**Comment by RemiLehe** [comment]
@PrometheusPi Regarding your remarks on the number of zeros, I was not sure if you are asking me to remove that zero, or if it was a personal note for yourself to remember to adapt your old scripts. Can you clarify ?

Regarding the particles for thetaMode, if it is okay with you I would still leave the file without particles, as it used in the openPMD-viewer tutorial, in order to show that the viewer can adapt to the absence of particles, in the GUI. But it is definitely not a very important point, so if you actually need the particles in the data of thetaMode, I can definitely add it. Please let me know.

**Comment by ax3l** [comment]
I think he means your "general scripts that read those files", e.g., our test scripts in `openPMD-viewer`? Also be aware that I just realized https://github.com/openPMD/openPMD-standard/issues/121 today for detecting the iteration from the file name.

So right now, using file names of the form `data_100.h5` and `data_50.h5` will likely be easier for reader implementations than padding zeros such as `data_00100.h5`. We can fix/refine this in `1.X`.

**Comment by ax3l** [comment]
After thinking once more about: it is not that much of a problem and we should just be more explicit that this _can_ be zero-padded.

**Comment by RemiLehe** [comment]
As far as I can tell, openPMD-viewer works fine with any number of zeros. (and reads the datasets of the PR correctly)
@ax3l If I understand you, I can leave the filenames as they are, is that correct ? If yes, you can probably merge the PR.

**Comment by ax3l** [comment]
absolutely, thank you a lot! :sparkles:

**Pull request merged**